changelog.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add plugin manager
 - Display current git branch of project in text view
 - Add menu to list git repository branches
+- Add search field for git branches menu
```

Comment:
to put at the end of it !

Reply:
like the previous pr, don't care, I will rework on the changelog before release version